tests/python/test_build_data.py

```
@@ -2099,6 +2099,26 @@ def test_normalize_place_page_enrichment_rejects_ui_display_name_and_review_desc
+        details.description = (
+            "Best ramen we've ever had. It helps that you get to make it yourself "
+            "(the noodles at least). Everything tastes better when you do it yourself! "
+            "Date day for a Saturday morning class. Great experience overall."
+        )
+
+        enrichment = build_data.normalize_place_page_enrichment(details)
+
+        self.assertIsNone(enrichment.description)
+
+        details.description = (
+            "That’s gotta be one of the best hot chocolate drink I’ve tasted in my life! "
```

Ruff (0.15.12)

[warning] String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)